htdocs/app/Helpers/scriptSaves_helper.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
```

Comment:
add a php doc comment here so we know what it does and how it works / what the parameter should be